slides/intro-extra-recipes.qmd

```diff
@@ -174,22 +175,22 @@ Trained methods need to calculate **sufficient information** to be applied again
 ## How to write a recipe
 
 :::{style="font-family: 'Source Code Pro', monospace; font-size: 0.8em;"}
-taxi_rec <- recipe(tip ~ ., data = taxi_train) %>%  
+forested_rec <- recipe(forested ~ ., data = forested_train) %>%  
 \ \ step_unknown(all_nominal_predictors()) %>%  
 \ \ step_dummy(all_nominal_predictors()) %>%  
 \ \ step_zv(all_predictors()) %>%  
-\ \ step_log(distance, offset = 0.5) %>%  
+\ \ step_log(canopy_cover, offset = 0.5) %>%  
```

Comment:
We should look into whether this transformation is actually helpful and choose another variable / step if not.